pagopa/io-app
# feat: [IOCOM-698] Add bottom sheet to `RemoteContentBanner` component

This PR depends on https://github.com/pagopa/io-app/pull/5294

## Short description
This PR adds bottom sheets to `RemoteContentBanner` component.

## List of changes proposed in this pull request
- updated the component `RemoteContentBanner` to display the bottom sheet
- updated locales

## How to test
Using `io-dev-api-server` generate a message with remote content. Open the message detail, tap on the link in the banner and check that the bottom sheet is displayed.
